examples/asr/conf/speech_multitask/fast-conformer_aed.yaml

```diff
@@ -13,6 +13,14 @@ name: "FastConformer-Transformer-MultiTask"
 #  may help (or even be required to) stabilize the training.
 init_from_nemo_model: null
 
+# If using example training script, below will be used to instantiate spl_tokens tokenizer.
+# Similar can be done by calling CanaryTokenizer.build_special_tokenizer(tokens, output_dir).
+# If a tokenizer exists in dir, will skip building and use tokenizer present in `tokenizer.langs.spl_tokenizer` instead.
+spl_tokens:
+  dir: ???
```

Comment:
The point of this is so users have an example case of setting up the Special Tokens. Training script uses this to create a Canary special tokens set so users can plug and play. Else we'll have multiple requests about setting up special tokens from people that are interested in just doing Canary.

Reply:
You need a way for users to disable this, not all know the ~ syntax to delete config and right now example script will always build tokenizer. 

Add a flag called build set to false by default

Reply:
The backend skips the build set if a tokenizer is present in directory. Can add the build flag but it would just be for force overwriting the tokenizer, which I don't think is safe for what's an example.